### PR TITLE
1919/qr code urls

### LINKED ISSUE #1919: Add qr code to enroll privacyIDEA Authenticator
In the HOTP, TOTP, PUSH enrollment page we could add a QR code, that links to the Play Store and App Store to install the privacyIDEA Authenticator.

How it would be added is to be discussed.
This should be configurable.

### PR DESCRIPTION
**Pull request by cornelinux**
See the commit comments of each commit

Closes #1919

### CODE REVIEW

File: privacyidea/static/components/token/views/token.enroll.hotp.html [hunk 1]
```diff
@@ -1,10 +1,30 @@
-<p class="help-block"
-   ng-hide="$state.includes('token.wizard')"
-   translate>
-The HOTP token is an event based token. You can paste a secret key or
-    have the server generate the secret and scan the QR code with a smartphone
-    app like the Google Authenticator turning your
-    smartphone into an authentication device.
+<p>
```
Comment (plettich) [inline privacyidea/static/components/token/views/token.enroll.hotp.html]: Can we make some kind of angular component (or whatever this is called) from this part so we don't have to edit all three parts?

Comment (cornelinux) [reply privacyidea/static/components/token/views/token.enroll.hotp.html]: No I would not want to do this.

File: doc/policies/webui.rst [hunk 2]
```diff
@@ -301,4 +301,47 @@ This way the text "Please sign in" on the login dialog can be changed. Since the
...
+will contain a QR code, that leads the user to the given URL.
+
+The idea is, that an organization running privacyIDEA can create its own URL,
+where the user is taken to, e.g.
+
+* Show information about the used Authenticator apps...
+* Do a device identification and automatically redirect the user to Google Play Store
+  or Apple App Store. Thus only need the user to show *one* QR code...
+* If an organization has it's own customized app or chooses to use another app, lead
+  the user to another App in the Google Play Store or Appe App Store.
```
[the remark this answers is not in the document]
Comment (cornelinux) [reply doc/policies/webui.rst]: Done

File: privacyidea/static/components/token/views/token.enroll.hotp.html (hunk 1 above)
Comment (plettich) [reply privacyidea/static/components/token/views/token.enroll.hotp.html]: ok

**Pull request merged**